src/components/Training/Modal.js

```diff
@@ -93,11 +93,11 @@ const Modal = ({ content, toggleModal }) => (
               <Padding bottom={2}>
                 <SectionTitle>{content.name}</SectionTitle>
```

Comment:
Worth finding out from designers whether this title should be restricted in width. See page pictures above, where "Our training" and "approach" are on different lines.

Remember this [FixedWidth](https://github.com/yldio/yld.io/pull/95/commits/aae5255bf65db79c3911738027642f9304513c8e) pattern I was talking about? This is usually intentional on designers' part - they do this so that typography is more visually pleasing and/or reads more easily.

Reply:
That pattern may fall down if the copy changes :/

The design specs don't mention anything regarding a maximum width and I checked with Antonas and it does span throughout the available space inside the container